themes/bootstrap3/templates/RecordDriver/DefaultRecord/data-publicationDetails.phtml

```diff
@@ -8,7 +8,7 @@
   <?php endif; ?>
   </span>
   <?php $pubDate = $field->getDate(); if (!empty($pubDate)): ?>
-    <span property="publicationDate"><?=$this->escapeHtml($pubDate)?></span>
+    <span property="datePublished"><?=$this->escapeHtml($pubDate)?></span>
```

Comment:
Since this fix is already merged to master, I tried to update your branch by merging in the latest master... but once again, I don't have write permission, so I wasn't able to push the update to you.

Reply:
Sorry for that... i pulled in the meantime. Still don't now what settings to change to make it easier next time.

Reply:
When you open a pull request, there should be a checkbox called "Allow edits from maintainers." If you check that, it should allow me to make edits... but maybe you're already doing that, and it's not working for some other reason.

Reply:
I remember enabling the checkbox a while ago, but right now it seems like this checkbox is not shown to me anymore, so i can't enable it. Maybe it has something to to with the way i work with git?

Basically what i am doing now is:
- git clone our local tuefind repository (as "origin")
- git add remote vufind.org/vufind (as "upstream")
- git checkout -b <upstream_schema_org (new branch)> upstream/master
- git add / commit files to local branch upstream_schema_org
- git push origin <upstream_schema_org>
- Then i send you a pull request

Do you know if one of these steps might cause the problem?

Reply:
That seems like a reasonable process to me. I wonder if there is some admin setting on the overall Tuefind repository that disallows external commits and overrides the pull request option.